assets/model_monitoring/components/model_monitor/model_data_collector_preprocessor/spec.yaml

```diff
@@ -4,7 +4,7 @@ type: spark
 name: model_data_collector_preprocessor
 display_name: Model Data Collector - Preprocessor
 description: Filters the data based on the window provided.
-version: 0.3.9
+version: 0.3.10
```

Comment:
Do you need to upgrade other components version?

Reply:
yes, but I'd like to do it in a separate PR and ask all component owners to sign off